hosts/mkSystemConfig.nix

```diff
@@ -103,7 +103,12 @@ in
             inherit hostName;
           };
         };
-        modules = [ "${path}/hosts/${hostName}/configuration.nix" ] ++ sharedModules;
+        modules =
+          if hostname == "OpenIris" then
+            [ "${path}/hosts/${hostName}/configuration.nix" ] ++ sharedModules ++ [ inputs.zapret.nixosModules.zapret ]
+          else
+            [ "${path}/hosts/${hostName}/configuration.nix" ] ++ sharedModules;
+
```

Comment:
No need for that,
Bloats the mkSystemConfig.nix 
Just add it as a module in your imports where you want to define that module, preferably the separate file like zapret.nix that you would import in your configuration.nix
```suggestion
```